cms/templates/cms/home_page.html

```diff
@@ -160,5 +177,90 @@
   </div>
 
 </div>
+
+<!-- Modal subscribe alerts -->
+<div id="subscribeAlert" class="modal fade" role="dialog">
```

Comment:
It's OK for now, but we should feel free to refactor the HTML coming from Nicolae.